docs/faq.rst

```diff
@@ -23,8 +23,15 @@ and then add ``permalink: /404.html`` in the `YAML front matter`_.
 
 If you are using the github provided domain, make sure to set the :confval:`notfound_urls_prefix` to your repository's name in between two forward slashes. For example if your repository is named ``MyRepo``, then ``notfound_urls_prefix = "/MyRepo/"``.
 
+Does this extension work with `Jupyter Book <https://jupyterbook.org/intro.html>`_?
+-------------------------------------------
+
+Yes.
+
+You will need to install the extension locally and add it to your Jupyter Book ``_config.yml`` as an extra extension, and you will likely need to set the :confval:`notfound_urls_prefix` to your repository's name in between two forward slashes. For example if your repository is named ``MyRepo``, then ``notfound_urls_prefix = "/MyRepo/"``.
 
-.. _YAML front matter: http://jekyllrb.com/docs/frontmatter/
+To see an example of a Jupyter Book ``_config.yml`` file that works with sphinx-notfound-page, see:
+https://github.com/melaniewalsh/Intro-Cultural-Analytics/blob/master/book/_config.yml
```

Comment:
What about using the commit hash so this is permanent?

```suggestion
https://github.com/melaniewalsh/Intro-Cultural-Analytics/blob/690ebe6/book/_config.yml
```

Reply:
Nice! Yeah good idea